packages/server/test/handler.spec.js

```diff
@@ -242,3 +254,76 @@ test('test access/claim provider', async () => {
   const result = await claim.execute(client)
   assert.deepEqual(result, [])
 })
+
+test('handle did:mailto audiences', async () => {
```

Comment:
btw it would be nice to have a Schema for `did:mailto` that matches with `did:mailto:${string}:${string}` because `DID.match({ protocol: 'mailto' })` does not satisfy `did:mailto:${string}:${string}`

Reply:
I agree, but that is probably out of scope for ucanto